libraries/Microsoft.Bot.Builder/ActivityHandler.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder
+{
+    public class ActivityHandler : IBot
```

Comment:
Well, ironically, that is what I originally called it. Feedback was call it ActivityHandler. FWIW I still think Bot is a better name. However, perhaps you should take this point up with @stevenic because I think the naming ActivityHandler sits better in the JavaScript world or something like that.

Reply:
The thing about making this abstract is that there are no abstract methods on it, so it seems odd to declare it as abstract, not that that seems like to big a deal.